src/main/resources/application.properties

```diff
@@ -0,0 +1,10 @@
+spring.datasource.url=${DB_URL}
```

Comment:
FYI Spring will automatically use environment variables if they are named the same way as properties, eg. SPRING_DATASOURCE_URL